src/backend/src/backend.cpp

```diff
@@ -6,8 +6,7 @@
 #include "mavsdk.h"
 #include "grpc_server.h"
 
-namespace mavsdk {
-namespace backend {
+using namespace mavsdk::backend;
```

Comment:
I don't understand this change. I can see how you can't have namespaces in C but you can't have the class either C?

Reply:
If I keep the namespace here, it gets ambiguous when building in C++. Because the header file says `struct MavsdkBackend*` and the implementation calls it `mavsdk::backend::MavsdkBackend*`. An alternative I considered was to `#ifdef __cplusplus` and define all the functions once for C and once for C++. I've seen examples that have a more complicated structure to avoid the `#ifdef`, too. I haven't seen something simple and elegant, though :confused:.

I could also do something like `#define MAVSDK_BACKEND_TYPE "struct MavsdkBackend*"` for C, and `#define MAVSDK_BACKEND_TYPE "mavsdk::backend::MavsdkBackend"` for C++, so that the functions look like:

```
DLLExport MAVSDK_BACKEND_TYPE runBackend();
```

But I'm not sure what's best. Opinions? :sweat_smile:

Reply:
You clearly thought more about this than me :smile:.

I would probably put the `#ifdef __cplusplus` around the namespace, I think that's fairly common.

Reply:
I tried and I failed so let's merge this as is for now.